llmstudio/tracking/tracker.py

```diff
@@ -18,5 +18,14 @@ def log(self, data: dict):
         )
         return req
 
+    def session(self, data: dict):
```

Comment:
in this case, we might need more methods:
1. to add something to the session
2. to get the session_history